prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -273,8 +273,10 @@ ASK_CTP_LUMI_GPU=
 local restOpt=
 while [[ $# -gt 0 ]]; do
   case "$1" in
-    --lumi-type=*) ASK_CTP_LUMI_GPU=" --lumi-type ${1#*=}"; [[ ${1#*=} == "2" ]] && NEED_TPC_SCALERS_WF=1; shift 1;;
-    --lumi-type) ASK_CTP_LUMI_GPU=" --lumi-type ${2}"; [[ ${2} == "2" ]] && NEED_TPC_SCALERS_WF=1; shift 2;;
+    --lumi-type=*) ASK_CTP_LUMI_GPU="${ASK_CTP_LUMI_GPU} --lumi-type ${1#*=}"; shift 1;;
+    --lumi-type) ASK_CTP_LUMI_GPU="${ASK_CTP_LUMI_GPU} --lumi-type ${2}"; shift 2;;
+    --corrmap-lumi-mode=*) ASK_CTP_LUMI_GPU="${ASK_CTP_LUMI_GPU} --corrmap-lumi-mode ${1#*=}"; shift 1;;
```

Comment:
Hello @shahor02 
I moved the `--corrmap-lumi-mode` to the `tpc-reco-workflow` and I also added it to `gpu-reco-workflow`. Otherwise the default value was actually used at some places in the code (in`void CorrectionMapsLoader::requestCCDBInputs(std::vector<InputSpec>& inputs, std::vector<o2::framework::ConfigParamSpec>& options, int lumiScaleType, int lumiScaleMode)` the `lumiScaleMode` was always `0`). I assume this is fine? I also added the `--corrmap-lumi-mode` to the `parse_TPC_CORR_SCALING`. Is this correct how I did it?

Reply:
Indeed, as it is now the `corrmap-lumi-mode` could not be used: as an option defining device inputs it must be defined on the workflow level. But then it should be added to all global tracking workflows using TPC corrections (same as `--lumi-type`). Let me open a PR on your branch to minimize the boiler plate code.
The fix for the dpl-workflow looks fine.

Reply:
@matthias-kleiner for a reason I don't understand I am not able to open PR on your branch, `matthias-kleiner/AliceO2` does not show up in the `Choose a Base Repository` menu...
Could you rebase your branch to my branch https://github.com/shahor02/AliceO2/tree/DCAd_monitoring (forked from yours with just 1 commit atted, you can also cherry-pick https://github.com/shahor02/AliceO2/commit/884ce9881878922f299171dbb544666123e8f1d4 )
https://github.com/shahor02/AliceO2/commit/884ce9881878922f299171dbb544666123e8f1d4